pkg/config/config.go

```diff
@@ -84,15 +123,27 @@ func Save(cfg *Config) error {
 	if err != nil {
 		return err
 	}
+
+	data, err := json.MarshalIndent(cfg, "", "  ")
+	if err != nil {
+		return fmt.Errorf("can't marshal config: %v", err)
+	}
+
+	keyring := os.Getenv("RH_KEYRING")
```

Comment:
@tylercreller How about a function `IsKeyRingManaged` (or something) that returns a `bool`. Means we have a single location that checks the value of the `RH_KEYRING` env variable.

Reply:
Good call-outs, thanks Rob. Updated.